lib/Collection.php

```diff
@@ -54,7 +54,7 @@ public function renderStyleTags() {
       /// @todo Fetch and minify proprietary styles.
 
       $tags .= '<link rel="stylesheet" type="text/css" '
-        . 'href="' . htmlspecialchars($style) . '"></link>' . "\n";
+        . 'href="' . htmlspecialchars($style) . '">' . "\n";
```

Comment:
"if the element is one of the void elements, or if the element is a foreign element, then there may be a single U+002F SOLIDUS character (/). This character has no effect on void elements, but on foreign elements it marks the start tag as self-closing"

http://dev.w3.org/html5/spec-author-view/syntax.html#syntax-start-tag

(See also:
http://stackoverflow.com/questions/10598501/closing-tags-in-html5
http://tiffanybbrown.com/2011/03/23/html5-does-not-allow-self-closing-tags/
)